File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.js
function solution(participant, completion) {
    let answer = '';
    const pArr = participant.sort()
    const cArr = completion.sort()
    
    for (let i = 0; i < pArr.length; i++){
        if (pArr[i] !== cArr[i]) 
            return pArr[i]
    }
}

File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.py
def solution(participant, completion):
    dict = {}
    
    for a in participant:
        dict[a] = dict.get(a, 0) + 1

    for b in completion:
        dict[b] -= 1
        if dict[b] == 0: del dict[b]

    return list(dict.keys())[0]
